common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1137,6 +1139,8 @@
                     .removeClass('is-active')
                     .find('.control-text')
                         .text(gettext('Turn on closed captioning'));
+
```

Comment:
I guess it comes down to what the analytics team wants the data to represent.  My assumption is that they are looking for an event any time a user tries to open the captions, regardless of the success or failure of the actual method.  In that case, you would want to fire the event as soon after the user's action as possible.  Might be a good question for @stroilova ?

This particular method isn't that complicated, so it probably doesn't make much of a difference, but would be good to know.